### PR TITLE
VIITE-2204 removed filter that ignore road addresses that are under c…

### PR DESCRIPTION
**Pull request by Campanha**
…onstruction

### CODE REVIEW

File: viite-UI/src/model/RoadCollection.js [hunk 1]
```diff
@@ -171,7 +171,7 @@
           });
 
         var nonUnderConstructionRoadLinkGroups = _.reject(roadLinkGroups, function(group) {
```
Comment (skosonen) [inline viite-UI/src/model/RoadCollection.js]: Should this variable name be changed since this is not "nonUnderConstructionRoadLinkGroups" any more?

Comment (Campanha) [reply viite-UI/src/model/RoadCollection.js]: true

Comment (skosonen) [reply viite-UI/src/model/RoadCollection.js]: Also good to make sure that on line 177 we don't concatenate same links multiple times.

Comment (Campanha) [reply viite-UI/src/model/RoadCollection.js]: yes we dont get same links since they are being filtered beforehand

**Pull request merged**